lib/integrations/calendar/CalendarManager.ts

```diff
@@ -111,9 +111,12 @@ export const getBusyCalendarTimes = async (
     .map((credential) => getCalendar(credential))
     .filter(notEmpty);
 
-  const results = await Promise.all(
-    calendars.map((c) => c.getAvailability(dateFrom, dateTo, selectedCalendars))
-  );
+  let results: EventBusyDate[][] = [];
```

Comment:
In case that any of the integrations fail we let the user create an event and we log the error.